pandas/tests/plotting/test_frame.py

```diff
@@ -204,6 +204,23 @@ def test_color_and_style_arguments(self):
         with pytest.raises(ValueError):
             df.plot(color=["red", "black"], style=["k-", "r--"])
 
+    def test_color_and_marker(self):
+        # GH21003 2018-05-10
+        df = DataFrame(np.random.random((7, 4)))
+        # combining a color string and a marker letter should be allowed
+        df.plot(color="green", style="d")  # d for diamond
+
+    def test_color_list_and_marker(self):
+        # GH21003 2018-04-23
+        df = DataFrame(np.random.random((7, 4)))
+        # combining a color list and a marker letter should be allowed
+        color = ["yellow", "red", "green", "blue"]
+        ax = df.plot(color=color, style="d")
+        output_colors = [line.get_color() for line in ax.lines]
+        assert output_colors == color  # each "line" is one of the four colors
+        # In the Github bug the color list was passed to MPL like this:
+        # ax.lines[i].get_color() == ['yellow', 'red', 'green', 'blue']
```

Comment:
"the Github bug" is a bit vague, and I'm not sure I see what value this comment adds

Reply:
I think this comment is useful so folks can comprehend why this test was written - i.e. how this used to go wrong. I'm working on a better formulation for line 221.